lldb/source/API/SBThread.cpp

```diff
@@ -1208,7 +1208,8 @@ bool SBThread::GetStatus(SBStream &status) const {
   ExecutionContext exe_ctx(m_opaque_sp.get(), lock);
 
   if (exe_ctx.HasThreadScope()) {
-    exe_ctx.GetThreadPtr()->GetStatus(strm, 0, 1, 1, true);
+    exe_ctx.GetThreadPtr()->GetStatus(strm, 0, 1, 1, true,
+                                      /*show_hidden*/ true);
```

Comment:
Nit: we use `=*/` because that's what the [clang-tidy check](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/argument-comment.html) understands. 
```suggestion
                                      /*show_hidden=*/ true);
```

Reply:
TIL!